### ISSUE TITLE
Add categories with a script

### ISSUE THREAD

**Issue by ooii** [bnjbvr/kresus#168]
Hi, 

I have a long list of categories that I'd like to use in Kresus but don't want to add them one by one from the web interface. 
Could someone tell me where they are stored and how I can add them? I'll write my script to do so.

Thanks.

**Comment by bnjbvr** [comment]
They're in the Cozy data-system, which is a couchdb (you can look at the data thanks to the Data Browser application). You could get ahead with a script in `server/models/bankcategories`, I think.
That being said, that sounds like a great idea, and making a form for adding several categories in a row shouldn't be too hard.

**Comment by nicofrand** [comment]
Maybe Kresus could create several categories if they are separated by a comma ? If I write "catA, CatB, anotherCat", this will create 3 categories (catA, CatB, anotherCat) ?

**Comment by bnjbvr** [comment]
Yes, the separator is a technical detail :) I think we'll need a new form section though, as having a single form field for two different interactions feel wrong (and would need an helper text for explaining the advanced interaction, that is adding several categories in a row).

**Comment by ooii** [comment]
Do you have any plan on adding this form (since it's a great idea:))? I couldn't manage to find where to put a script to populate the database.

**Comment by bnjbvr** [comment]
Yes, I'll probably do that, it shouldn't be too much work (a line or two to change in the server, some more in the client). I can't guarantee it'll be in the next release (today or tomorrow) though.

**Comment by ooii** [comment]
One more feature: set categories color.

**Comment by ooii** [comment]
Hi, any news on that?
Thanks.

**Comment by bnjbvr** [comment]
Nope, not yet. It's a bit low priority to me, still very much willing to help if somebody wants to implement this. For what it's worth, you could use the export format to add a list of categories, by looking at how the export file is formatted it shouldn't be too hard; still, there's no automated way to do this.
